README.md

```diff
@@ -1,48 +1,55 @@
+7. A proposal will be accepted if it receives +2 (including the proposer) and no vetoes (-1) or has not received enough votes in 10 business days.
 8. If a proposal is vetoed, and it cannot be revised to satisfy all parties, then it can be resubmitted for an override vote in which a majority of all eligible voters indicating +1 is sufficient to pass it. Note that this is a majority of all committee members, not just those who actively vote.
 9. Upon completion of discussion and voting the proposer should announce whether they are proceeding (proposal accepted) or are withdrawing their proposal (vetoed).
 10. The Chair gets a vote.
 11. The Chair is responsible for keeping track of who is a member of the Project Steering Committee.
 12. Addition and removal of members from the committee, as well as selection of a Chair should be handled as a proposal to the committee. 
 13. The Chair adjudicates in cases of disputes about voting.
 
-## When is Vote Required?
+### When is Vote Required?
 
 Generally speaking, technical decisions should be made by consensus within openEO components. However, if a decision cannot be decided by consensus of the committers involved in the component, any committer can request the decision be brought to a vote of the openEO PSC. Technical decision that would be suitable for such a process include:
 
 * Any change to PSC membership (new members, removing inactive members)
-* Anything that could cause backward compatibility issues in API, process specifications or shared software components.
+* Anything that could cause backward compatibility issues in normative specifications (e.g., API or process specifications).
 * Adding or changing substantial amounts of the specifications or code.
 * Issues of procedure (e.g. changes to these guidelines).
 * When stable releases should take place.
 * Strategic decisions regarding the project and anything that might be controversial.
 
-## Observations
+### Final remarks
 
 * The Chair is the ultimate adjudicator if things break down.
 * The absolute majority rule can be used to override an obstructionist veto, but it is intended that in normal circumstances vetoers need to be convinced to withdraw their veto. We are trying to reach consensus.
-* Software components that at least two other software repositories under https://github.com/Open-EO depend on, must implement a review system for breaking changes (e.g. GitHub Pull Request Approvals) that require a certain number of approvals: 2 dependants require 1 approval, more than 2 dependants require 2 approvals.
+* Software components that at least two other software repositories under https://github.com/Open-EO depend on, must implement a review system for breaking changes (e.g. GitHub Pull Request Approvals) that require a certain number of approvals: 2 dependants require 1 approval, more than 2 dependants require 2 approvals. Exceptions for this must go through the PSC process.
```

Comment:
I guess there could be cases where there exceptions are okay, but that should be discussed in the PSC.